tests/e2e/sentinel/transitions/mark_for_outbound.js

```diff
@@ -75,12 +75,12 @@ const wipeTasks = () => getTasks()
   });
 
 describe('mark_for_outbound', () => {
-  afterEach(done => Promise.all([utils.revertSettings(), wipeTasks()]).then(done));
-  afterAll(done => utils.revertDb().then(done));
+  afterEach(() => Promise.all([utils.revertSettings(true), wipeTasks()]));
+  afterAll(() => utils.revertDb());
 
   describe('when external server is up', () => {
-    beforeEach(done => startMockApp().then(done));
-    afterEach(done => { stopMockApp(); done(); });
+    beforeEach(() => startMockApp());
```

Comment:
Make the same change on line 429 of this file

Reply:
done.